Makefile

```diff
@@ -24,6 +24,7 @@ clean-build:
 	rm -fr *.egg-info
 	rm -fr .eggs
 	rm -fr .cache
+	rm -r kolibri/dist/exercise_perseus_renderer || true
```

Comment:
I'm not sure either what's happening. According to the error, it's expecting `exercise_perseus_renderer` to be a git repo, but that's not the case.

I'll add a comment marking this as a code smell, so we can flag and fix this in the future.